python/tests/fixtures/build_memory_index.py

```diff
@@ -12,19 +11,15 @@
 
 
 def build_random_vectors_and_memory_index(
-        dtype,
-        metric,
-        with_tags: bool = False,
-        index_prefix: str = "ann",
-        seed: int = 12345
+    dtype, metric, with_tags: bool = False, index_prefix: str = "ann", seed: int = 12345
```

Comment:
Why move this back to one-line from multi-line?  Did the formatter do this?  Not a big deal, I just am curious about the style.

Reply:
yep, black did this for us.  I'm not keen on it personally, but that's the price you pay when you pick a code formatter to do it for you; it's going to make someone unhappy about something :(